src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp

```diff
@@ -321,10 +321,8 @@ namespace TerminalAppLocalTests
             { "keys": ["ctrl+c"], "command": { "action": "splitPane", "split": null } },
             { "keys": ["ctrl+d"], "command": { "action": "splitPane", "split": "vertical" } },
             { "keys": ["ctrl+e"], "command": { "action": "splitPane", "split": "horizontal" } },
-            { "keys": ["ctrl+f"], "command": { "action": "splitPane", "split": "none" } },
             { "keys": ["ctrl+g"], "command": { "action": "splitPane" } },
-            { "keys": ["ctrl+h"], "command": { "action": "splitPane", "split": "auto" } },
-            { "keys": ["ctrl+i"], "command": { "action": "splitPane", "split": "foo" } }

```

Comment:
Just double-checking, you're removing some of these tests because they should be covered by the JsonUtils tests?

Reply:
yeah, they were illegal.